utils/helpers.js

```diff
@@ -0,0 +1,283 @@
+require('dotenv').config();
+
+function delay() {
+    return new Promise(resolve => setTimeout(resolve, process.env.RETRY_DELAY || 2000));
+}
+
+async function getBalance(erc20Contract, tokenAddress, signersAddress) {
+    const balance = await erc20Contract.balanceOf(tokenAddress, signersAddress);
+    return balance;
+}
+
+async function getCount(proxyContract) {
```

Comment:
Q: what is this. count of? Maybe add a method comment

Reply:
Done.

Reply:
The contract to update has a public variable called `count`. The test increments and decrements the variable.  Then upgrades the contract and again tests to confirm that the `count` can be incremented and decremented.  This approach leverages inheritance to make contracts upgradeable.  The new contract inherits the `count` from the original contract.